docs/source/cheatcodes.rst

```diff
@@ -0,0 +1,26 @@
+VM Cheatcodes within Titanoboa
+=====================================
+
+Developers who make frequent use of cheatcodes, from other frameworks such as foundry will be happy to found similar functionality within Titanoboa.
+
+Pranking
+--------
+
+In order to "prank" or impersonate an address simply call `boa.prank()`. The example below shows a simple prank as an `admin` account, to deploy a contract.
+
+.. code-block:: python
+    with boa.env.prank(admin):
+        return boa.load('contracts/testing/OptimizeMath.vy')
+..
+
+Everything within the `with` block will occur on behalf of the user being impersonated.
+
+Time-Travel
+-----------
+
+In order to advance the timestamp, boa exposes a `time_travel` method, which seconds passed in will advance the current `block.timestamp` by within the current environment.
+
+This example demonstrated advancing `block.timestamp` by 1 month.
+.. code-block:: python
+  boa.env.time_travel(86400 * 30)
+..
```

Comment:
it's a bit awkward to have these in the cheatcode section, plus i worry it overlaps with the docs for `boa.env`. but i think maybe it's ok for now